src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firstLeg * secondLeg / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: "
```

Comment:
The `draw()` method is conceptually different from the `getArea()` method and should not be in the same interface or abstract class. Consider separating them into different interfaces or abstract classes according to their responsibilities.